styles.css

```diff
@@ -0,0 +1,88 @@
+
+.images {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  position: relative;
+  margin-top: 40px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.time-wifi-battery {
+  width: 10px;
+  height: 12px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.wifi-battery {
+  float: right;
+  margin-right: 20px;
+}
+
+#headline {
+  width: 375px;
+  height: 700px;
+  margin: 0 auto;
+  background-image: url(./images/header-shapes\ mobile@2x.svg);
+  border-bottom-left-radius: 80px;
+}
+
+body.active {
+  overflow-y: hidden;
+}
+
+.header {
+  background-image: url();
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: var(--header-color);
+  padding: 128px 24px 0;
+}
```

Comment:
- [ ] Kindly make sure your design looks as much as possible as the Figma template. Check the font family, font size and font color.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/185637436-e77ccdd9-6844-4486-b0f9-0cf85664bdca.PNG)
Figma template
![figma](https://user-images.githubusercontent.com/85095702/185637482-53746d16-5e4f-4ef5-9e86-417d9ce64d7d.PNG)

- [ ] Also make sure there is no horizontal scroll on your page. I will suggest that when adding margins, paddings, width, and height to your page, always favor the use of percent-based units (vw,vh,%) over the fixed unit (px) as this will help improve the responsiveness of your page.

Reply:
ok i will try this!